History/HistoryEntry.h

```diff
@@ -61,9 +61,86 @@ struct HistoryInlineRef
     bool        mIsRemote   : 1;
     bool        mIsTag      : 1;
     bool        mIsStash    : 1;
+    bool        mIsDetached : 1;
+
+    bool operator ==(const HistoryInlineRef& other) const
```

Comment:
Answer: Try removing this and write something like `if ( refA == refB ) ...`. The compiler fails in this case. Or am I missing something?